apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx

```diff
@@ -36,9 +41,14 @@ const ConnectedVideoStep = (props: ConnectedAppStepProps) => {
                     <AppConnectionItem
                       type={item.type}
                       title={item.name}
+                      isDefault={item.slug === metadata?.defaultConferencingApp?.appSlug}
                       description={item.description}
                       logo={item.logo}
+                      slug={item.slug}
                       installed={item.userCredentialIds.length > 0}
+                      defaultInstall={
+                        item.appData?.location?.linkType === "dynamic" && !hasAnyInstalledVideoApps
```

Comment:
We need to consider only `dynamic` linkType apps here. Otherwise if someone installs, say AroundVideo first and then installs Zoom, zoom won't be considered `default` because  `hasAnyInstalledVideoApps` would be true at that time.